packages/govuk-frontend/src/govuk/init.jsdom.test.mjs

```diff
@@ -228,17 +228,41 @@ describe('createAll', () => {
   })
 
   class MockComponent extends GOVUKFrontendComponent {
-    constructor(...args) {
-      super(...args)
-      this.args = args
+    /**
+     * @param {Element | null} $root - HTML element to use for accordion
+     * @param {...MockConfig} [config] - Accordion config
+     */
+    constructor($root, ...config) {
+      super($root, ...config)
+      this.args = [$root, ...config]
     }
 
     static checkSupport() {
       GOVUKFrontendComponent.checkSupport()
     }
 
     static moduleName = 'mock-component'
+
+    static schema = Object.freeze({
+      properties: {
+        attribute: { type: 'string' },
+        __test: { type: 'boolean' }
+      }
+    })
+
+    /**
+     * @type {MockConfig}
+     */
+    static defaults = Object.freeze({
+      attribute: '',
+      __test: false
+    })
```

Comment:
**suggestion** Is there a way the `GOVUKFrontendComponent` class can ensure these objects get frozen, allowing people to just set regular objects (maybe through a setter that freezes whatever it receives)? Is it even necessary?